test/api_test.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+import json
+
+import configparser
+
+from glusterapi import Client
+
+
+class TestPeerInfo(object):
```

Comment:
you can rename it as glusterdConfig as it contains the host and devices list

Reply:
done